fluss-server/src/main/java/com/alibaba/fluss/server/replica/ReplicaManager.java

```diff
@@ -851,7 +857,7 @@ private void addFetcherForReplicas(
                     bucketAndStatus.put(
                             tb,
                             new InitialFetchStatus(
-                                    tb.getTableId(), leader.get(), logTablet.localLogEndOffset()));
+                                    tb.getTableId(), leaderId, logTablet.localLogEndOffset()));
```

Comment:
As we don't need to get the leader `ServerNode` here, is it still necessary to return a bucket level error to CoordinatorServer when the leader `ServerNode` is not in metadata?

Reply:
Yes, I believe we don't need to return a bucket level error . I meant to remove bucket level error thrown in #644 since #644 is part of caused by the thrown error.  But I think we can remove bucket level error thrown in here, and along with #664,  #644 won't happen.